neo/SmartContract/Manifest/ContractGroup.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Neo.Cryptography;
+using Neo.Cryptography.ECC;
+using Neo.IO.Json;
+
+namespace Neo.SmartContract.Manifest
+{
+    /// <summary>
+    /// A group represents a set of mutually trusted contracts. A contract will trust and allow any contract in the same group to invoke it, and the user interface will not give any warnings.
+    /// A group is identified by a public key and must be accompanied by a signature for the contract hash to prove that the contract is indeed included in the group.
+    /// </summary>
+    public class ContractGroup
+    {
+        /// <summary>
+        /// Pubkey represents the public key of the group.
+        /// </summary>
+        public ECPoint PubKey { get; set; }
+
+        /// <summary>
+        /// Signature is the signature of the contract hash.
+        /// </summary>
+        public byte[] Signature { get; set; }
+
+        /// <summary>
+        /// Parse ContractManifestGroup from json
+        /// </summary>
+        /// <param name="json">Json</param>
+        /// <returns>Return ContractManifestGroup</returns>
+        public static ContractGroup FromJson(JObject json)
+        {
+            return new ContractGroup
+            {
+                PubKey = ECPoint.Parse(json["pubKey"].AsString(), ECCurve.Secp256r1),
```

Comment:
Maybe in neo 3 we should use the first byte for parse the curve. That allow us in the future to choose better cryptographic methods

Reply:
We can do it later.